manifests/plugin/python/module.pp

```diff
@@ -0,0 +1,34 @@
+# Single module definition
+define collectd::plugin::python::module (
+  $config,
+  $script_source = undef,
+  $modulepath    = $collectd::plugin::python::modulepath,
+  $module        = $title,
+  $ensure        = 'present',
+  $order         = '50',
+  $python_conf   = "${collectd::params::plugin_conf_dir}/11-python-config.conf",
+){
+  include collectd::params
+  include collectd::plugin::python
+
+  validate_hash($config)
+
+  if $script_source {
+    file { "${module}.script":
+      ensure => $ensure,
+      path   => "${modulepath}/${module}.py",
```

Comment:
A path like `puppet:///modules/myorg/elasticsearch_collectd_python.py` won't pass through `validate_absolute_path()`. Is there some other function that allow paths starting with `puppet://`?